examples/gaussian_process/plot_gpr_prior_posterior.py

```diff
@@ -24,11 +24,13 @@
 
 
 kernels = [1.0 * RBF(length_scale=1.0, length_scale_bounds=(1e-1, 10.0)),
-           1.0 * RationalQuadratic(length_scale=1.0, alpha=0.1),
+           1.0 * RationalQuadratic(length_scale=1.0, alpha=0.1,
+                                   alpha_bounds=(1e-5, 1e6)),
```

Comment:
I am still getting the same number of convergence warnings with this change. Lets revert this for now and keep the other change?